### PR TITLE
Quell some warnings in static analyzers

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/core/props/phaseUsageFromDeck.cpp [hunk 1]
```diff
@@ -135,7 +135,7 @@ PhaseUsage phaseUsage(const Phases& phases)
     pu.has_zFraction = phases.active(Phase::ZFRACTION);
     if (pu.has_zFraction) {
         pu.phase_pos[BlackoilPhases::ZFraction] = numActivePhases;
-        ++ numActivePhases;
+        //++ numActivePhases;
```
Comment (joakim-hove) [inline opm/core/props/phaseUsageFromDeck.cpp:138]: If it is unnecessary it is unnecessary - delete!

Comment (bska) [reply opm/core/props/phaseUsageFromDeck.cpp:138]: > If it is unnecessary it is unnecessary - delete!

This one stays.  I don't care if it's unnecessary or if static analyzers warn about it.  It makes the code easier to maintain if we add new enumerators to `BlackoilPhases`.

Comment (akva2) [reply opm/core/props/phaseUsageFromDeck.cpp:138]: so you want me to drop the commit?

Comment (bska) [reply opm/core/props/phaseUsageFromDeck.cpp:138]: Just to be clear: I want this particular change reverted and the increment reinstated.

Comment (bska) [reply opm/core/props/phaseUsageFromDeck.cpp:138]: Just to be clear: I want this particular change reverted and the increment reinstated.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**